documentation/examples.md

```diff
@@ -92,28 +92,26 @@ D('example.com', REG, DnsProvider('R53'),
 )
 ```
 
-## Add comments along complex SPF records
-
-You normally can't put comments in the middle of a string,
-but with a little bit of creativity you can document
-each element of an SPF record this way.
+## Use SPF_BUILDER to add comments along SPF records
```

Comment:
Suggestion: Add some explanation like...

SPF records are usually very long strings without any comments.  Over time it may become
difficult to remember why a particular item was included.  
With `SPF_BUILDER`, you can easily annotate each part of the SPF record.